### ISSUE TITLE
Investigate tuning of keepalive & hole punching on mobile platforms

### ISSUE THREAD

**Issue by glimberg** [zerotier/ZeroTierOne#1618]
Investigate tuning  UDP hole punching on Android/iOS a bit.  I don't imagine using ZeroTier for incoming connections is a huge use case on these platforms.  It's more from device to external location.

This would likely require an option to make keepalives/hole punching work like the full version of ZeroTier as well, but defaulting to assume outgoing connections only should work fine for the vast majority of users on a mobile device, and would likely save a ton of battery life.

**Comment by erikh** [comment]
This could be a configuration toggle for other classes of devices in the runtime if you did it right, e.g., "priority outbound" connections.

**Comment by glimberg** [comment]
Agreed.  Definitely could be useful as a tunable parameter for all platforms, too.
